domains/transfer/TransferAgreementAutomation/Worker.IntegrationTests/TransferAutomationApplicationFactory.cs

```diff
@@ -60,8 +69,21 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
         builder.UseSetting("TokenValidation:PublicKey", publicKeyBase64);
         builder.UseSetting("TokenValidation:Issuer", "demo.energioprindelse.dk");
         builder.UseSetting("TokenValidation:Audience", "Users");
-        builder.UseSetting("TransferApi:Url", "localhost:5001");
-        builder.UseSetting("TransferApi:Version", "20231123");
+
+        var connectionStringBuilder = new DbConnectionStringBuilder
+        {
+            ConnectionString = postgresContainer.ConnectionString
+        };
+        builder.UseSetting("Database:Host", (string)connectionStringBuilder["Host"]);
+        builder.UseSetting("Database:Port", (string)connectionStringBuilder["Port"]);
+        builder.UseSetting("Database:Name", (string)connectionStringBuilder["Database"]);
+        builder.UseSetting("Database:User", (string)connectionStringBuilder["Username"]);
+        builder.UseSetting("Database:Password", (string)connectionStringBuilder["Password"]);
+
+        builder.ConfigureTestServices(services =>
+        {
+            services.Remove(services.First(s => s.ImplementationType == typeof(TransferAgreementsAutomationWorker)));
```

Comment:
You create a fixture of the startup class, to then remove it? Why?

Reply:
I only remove the worker. It is not used for any integration tests. I remove if because it made our controller tests flaky